### ISSUE TITLE
Plugin insists on removing empty line between imports and a semicolon (;)

### ISSUE THREAD

**Issue by IlyaSemenov** [lydell/eslint-plugin-simple-import-sort#12]
The plugin insists on removing the delimiting empty line between the imports and a semicolon. This affects, for example, inline function calls (which are prepended with a semicolon by prettier).

Example:

```ts
import { foo } from "bar"

;(async function() {
  await foo()
})()
```

See screenshots:

<img width="201" alt="Screenshot 2019-05-09 at 13 13 03" src="https://user-images.githubusercontent.com/128121/57431262-6f37db00-725c-11e9-9d52-eef411766f2e.png">

<img width="525" alt="Screenshot 2019-05-09 at 13 13 13" src="https://user-images.githubusercontent.com/128121/57431264-71019e80-725c-11e9-9dbc-8f40d2a74e96.png">

The "fixed" version:

<img width="197" alt="Screenshot 2019-05-09 at 13 13 17" src="https://user-images.githubusercontent.com/128121/57431266-7363f880-725c-11e9-94e1-cab6daaebc3b.png">

### Expected behaviour

Allow empty line between imports and a block starting with a semicolon (as on screenshot 1).

If you need a reproduction repo I can create one. But I think the problem will repeat if you just copy/paste my example to any environment.

**Comment by lydell** [comment]
Good catch, I didn’t think about semicolon-less JS style.

**Comment by lydell** [comment]
Fixed in v3.1.1.
